tool/metrics-compose/docker-compose.yml

```diff
@@ -52,7 +52,7 @@ services:
       - "-p2p.metrics"
       - "-node.no-notif"
       - "-node.listeners=/ip4/0.0.0.0/tcp/9091"
-      - "-p2p.swarm-listeners=/ip4/0.0.0.0/tcp/6001,/ip4/0.0.0.0/udp/6001/quic"
+      - "-p2p.ipfs-listeners=/ip4/0.0.0.0/tcp/6001,/ip4/0.0.0.0/udp/6001/quic"
```

Comment:
hum, if you need this, then your image is outdated, please run `docker-compose pull` on your testing host